tests/components/image_processing/test_microsoft_face_detect.py

```diff
@@ -8,15 +8,16 @@
 import homeassistant.components.microsoft_face as mf
 
 from tests.common import (
-    get_test_home_assistant, assert_setup_component, load_fixture, mock_coro)
+    get_test_home_assistant_http, assert_setup_component, load_fixture,
+    mock_coro)
 
 
 class TestMicrosoftFaceDetectSetup(object):
     """Test class for image processing."""
 
     def setup_method(self):
         """Setup things to be run when tests are started."""
-        self.hass = get_test_home_assistant()
+        self.hass = get_test_home_assistant_http()
```

Comment:
If it's not going to use http in the tests, we should use `mock_http_component`. No need to bind a port and whatnot.